rdflib/plugins/serializers/turtle.py

```diff
@@ -292,122 +294,125 @@ def getQName(self, uri, gen_prefix=True):
             return False
-        self.write('\n' + self.indent() + '[]')
+        self.write("\n" + self.indent() + "[]")
         self.predicateList(subject)
-        self.write(' .')
+        self.write(" .")
         return True
 
     def path(self, node, position, newline=False):
-        if not (self.p_squared(node, position, newline)
-                or self.p_default(node, position, newline)):
-            raise Error("Cannot serialize node '%s'" % (node, ))
+        if not (
+            self.p_squared(node, position, newline)
+            or self.p_default(node, position, newline)
+        ):
+            raise Error("Cannot serialize node '%s'" % (node,))
 
     def p_default(self, node, position, newline=False):
         if position != SUBJECT and not newline:
-            self.write(' ')
+            self.write(" ")
         self.write(self.label(node, position))
         return True
 
     def label(self, node, position):
         if node == RDF.nil:
-            return '()'
+            return "()"
         if position is VERB and node in self.keywords:
             return self.keywords[node]
         if isinstance(node, Literal):
             return node._literal_n3(
                 use_plain=True,
-                qname_callback=lambda dt: self.getQName(
-                    dt, _GEN_QNAME_FOR_DT))
+                qname_callback=lambda dt: self.getQName(dt, _GEN_QNAME_FOR_DT),
+            )
         else:
             node = self.relativize(node)
 
             return self.getQName(node, position == VERB) or node.n3()
 
     def p_squared(self, node, position, newline=False):
-        if (not isinstance(node, BNode)
-                or node in self._serialized
-                or self._references[node] > 1
-                or position == SUBJECT):
+        if (
+            not isinstance(node, BNode)
+            or node in self._serialized
+            or self._references[node] > 1
+            or position == SUBJECT
+        ):
             return False
 
         if not newline:
-            self.write(' ')
+            self.write(" ")
 
         if self.isValidList(node):
             # this is a list
-            self.write('(')
+            self.write("(")
             self.depth += 1  # 2
             self.doList(node)
             self.depth -= 1  # 2
-            self.write(' )')
+            self.write(" )")
         else:
             self.subjectDone(node)
             self.depth += 2
             # self.write('[\n' + self.indent())
-            self.write('[')
+            self.write("[")
             self.depth -= 1
             # self.predicateList(node, newline=True)
             self.predicateList(node, newline=False)
             # self.write('\n' + self.indent() + ']')
-            self.write(' ]')
+            self.write(" ]")
             self.depth -= 1
 
         return True
 
-    def isValidList(self, l):
+    def isValidList(self, l_):
```

Comment:
Just curious, why wasn't just `l` OK here? (And if it isn't, the docsstring ought to correspond.)

Reply:
Only because `flake8` threw up a lot of warnings about possible variable ambiguity for `l` so there must be a higher-order variable `l` still within scope that this might clash with.

I suppose I should have thought o rebuild docstrings before putting in the PR. I'll try and do that and add to this.

Reply:
the flake8 error regarding `l` is because it can look like an uppercase i or a number 1. Its generally considered using a single letter variable named `l` is a bad thing. This solution works to suppress the error, and I think doesn't harm anything.

Reply:
I see. In practise I think `l` is good enough (in spite of flake8), but following that general advice, I think `lst` would be better, or even if we could establish a naming pattern like `rlist` (and similarly one for types, like `rtype`). (Going for `rdf_list` and `rdf_type` may seem even better/clearer, unless it becomes unclear whether they would reference an object or the terms themselves (`RDF.type` and `RDF.List`).)

Just something to consider going further, nothing critical for this PR.